e2e/app/page/workspace-about-page.ts

```diff
@@ -69,10 +69,10 @@ export default class WorkspaceAboutPage extends WorkspaceBase {
   }
 
   // if the collaborator is already on this workspace, just remove them before continuing.
-  async removeCollab(): Promise<void> {
-    const accessLevel = await this.findUserInCollaboratorList(config.collaboratorUsername);
+  async removeCollaborator(name = config.collaboratorUsername): Promise<void> {
```

Comment:
Make func more flexible and avoid Share modal if collaborator is not found.

Reply:
Yes, you're right. I added new func parameter for the collaborator email. Share modal is not opened when collaborator is not found in About page Collaborator List.